src/main/java/uk/gov/companieshouse/api/accounts/service/impl/SmallFullService.java

```diff
@@ -49,6 +49,9 @@
 
     private CompanyService companyService;
 
+    @Autowired
```

Comment:
Constructor is already autowired, which is how we should be doing it everywhere anyway, so this field ought to be not autowired AND be passed into the autowired constructor.

Reply:
I did initially put the StatementService in the constructor but on start up was getting a cyclic error which I couldn't quite sort. I will take another look

Reply:
I seem to remember Brad saying that this was a reason to do it this way - that it shows up this kind of issue.